source/docs/articles/sites/domains.md

```diff
@@ -64,7 +61,7 @@ Using the provided destinations in the Site Dashboard, create the recommended DN
 To serve your site from the bare domain, you must:
 
 1. Select a DNS provider that supports CNAME flattening, such as [CloudFlare (recommended)](https://support.cloudflare.com/hc/en-us/articles/200169056-CNAME-Flattening-RFC-compliant-support-for-CNAME-at-the-root), [ClouDNS](https://www.cloudns.net/features/), or [NameCheap](https://www.namecheap.com/domains/freedns.aspx).
-2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for **both** the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
+2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for both the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
```

Comment:
change to 
"... create two CNAME records. 
``` 
CNAME @ live-yoursite.pantheon.io
CNAME www live-yoursite.pantheon.io
```
"
This could use an image.